src/components/CommentForm/CommentForm.tsx

```diff
@@ -426,39 +421,37 @@ export const CommentForm = ({
                     onFocus={() => setIsActive(true)}
                 />
                 <div className={bottomContainer}>
-                    <div className={buttonContainerStyles}>
-                        <Button
-                            type="submit"
-                            size="small"
-                            data-testid="post-comment"
-                        >
-                            <div className={buttonTextStyles}>
+                    <Row>
+                        <>
```

Comment:
unnecessary

Reply:
It's needed because Row expects an element, not an element or string

Reply:
we could use `React.ReactNode` rather  than `JSX.Element | JSX.Element[]` in Row then to avoid having to do this.

I found a great answer on these variations here: https://stackoverflow.com/questions/58123398/when-to-use-jsx-element-vs-reactnode-vs-reactelement

Reply:
I wouldn't consider this essential for the PR through, so feel free to ignore

Reply:
Nice. That's interesting. I'll look into this separately